src/reducer/login.js

```diff
@@ -0,0 +1,27 @@
+import {LOGIN_USER_REQUEST,LOGIN_USER_SUCCESS,LOGIN_USER_FAILURE} from '../constants/user';
+import {initialState} from './user';
+
+const loginReducer = (state = initialState, action) => {
+    switch (action.type) {
+      case LOGIN_USER_REQUEST:
+        return {
+          ...state,
+          login_loading: true
```

Comment:
Why snake cases and login User shouldn't return list of users back. That's wrong. 
when user logins, the backend should return only one user back.